### DISCUSSION TITLE
Heading knob pull to manual controle doesnt work anymore

### DISCUSSION THREAD

**Discussion by beep747** [flybywiresim/aircraft#5489] [Help] [answered]
I'm at the latest dev version and at de latest update/hot fix of msfs 2020. As te title says: clicking the down button on the heading knob doesn't work anymore, which makes approach for landing impossible for me.
Anyone else with the same problem?
Jos

**Comment by Scottbels2020** [comment]
Yup, same problem  - I also keep losing autopilot, but understand that maybe frames related?

**Comment by mpondal** [comment]
It also happend to me in all knob that allows pull: Speed, High, STD presure . I do not see the way to pull any button.

**Reply by yuriks59** [reply]
Good health to all! These updates create problems for the smooth operation of the simulator. It is not clear why the buttons and toggle switches were illuminated when the mouse cursor hovered over them, now you have to suffer by switching them. The toggle switches for controlling the headlights and strobes are mixed up, you will not understand how to turn them on. The center and left headlights on the front landing gear are not lit at all. The hints had to be disabled due to their massive size.

**Reply by yuriks59** [reply]
I managed to switch by pressing the left mouse button and the wheels at the same time.

**Comment by tracernz** [comment, marked as answer]
Only the dev version is supported with Sim Update 5, and the heading modes all work fine with it. Please make sure you uninstall the marketplace version through the content manager. In legacy interaction mode you can push/pull the knobs with left click in the right spot, and with the new lock mode you can use side mouse buttons to push or pull the knob.

**Reply by beep747** [reply]
I'm always using the dev version and I  activated the legacy mode. Left click in the right spot but I don't find this right spot anymore. That's my problem,

**Reply by Scottbels2020** [reply]
Thanks for the great tips. I am able to use the side mouse buttons to operate the push pull on the development version successfully.
Any ideas why Auto-throttle may not be working for me consistently?

**Reply by beep747** [reply]
Is this answered? How? The clicks spots on the heading knob are gone (legacy mode). I'll try a fresh install of the aircraft  and see if this helps.

**Reply by Scottbels2020** [reply]
I used the side mouse buttons, not the legacy clicking mode. All seems good, except Auto-throttle keeps switching out!! - not good on a final approach!!!

**Reply by CptBayer** [reply]
I can´t uninstall via content manager?! How is it working?

![grafik](https://user-images.githubusercontent.com/76531893/128420547-5a91590e-ea11-405f-ba73-124d47454250.png)

**Comment by whiggy21** [comment]
Same Problem for me. Sel HDG Mode stoped working.

**Reply by NIX-CRO** [reply]
For me the same, I can manipulate all other knobs, but can't exit nav mode to hdg in flight, seems to work on the ground.

**Comment by guimarq97** [comment]
You need to switch in settings the input of the knobs, i think since the SU5 is some sort of compatibility patch for the consoles, they switched some of the inputs in some buttons in a way that makes it easier for console players to play, but i realized that broke some knobs for pc players. idk exactly whats the option called but i fixed my game by switching the input mode to the old one.

**Reply by yuriks59** [reply]
Thank you, I found how to configure everything and now the cabin works as it should.
